### PR TITLE
polish up the changes of @psav and prepare for build automation

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
this pr takes the fabulous work @psav did to make the automation work on small scale and prepares it for the image build automation
in addition it gets rid of the super-seeded older images and steam-lines the directory structure

### CODE REVIEW

File: Dockerfile.stable [hunk 1]
```diff
@@ -20,15 +24,20 @@ RUN mkdir -p /root/chrome-driver &&\
 ADD ./xstartup.sh /xstartup.sh
 RUN chmod 775 /xstartup.sh
 # selenium
-ADD http://selenium-release.storage.googleapis.com/$SELENIUM_VERSION/selenium-server-standalone-$SELENIUM_VERSION.0.jar \
+# TODO: make better url scheme
+ADD http://selenium-release.storage.googleapis.com/3.12/selenium-server-standalone-$SELENIUM_VERSION.jar \
```
Comment (mshriver) [inline Dockerfile.stable:28]: Why hardcode 3.12 instead of SELENIUM_VERSION?

File: Dockerfile.stable [hunk 2]
```diff
@@ -20,15 +24,20 @@ RUN mkdir -p /root/chrome-driver &&\
...
     /root/selenium-server/selenium-server-standalone.jar
+
 # firefox
-ADD https://download-installer.cdn.mozilla.net/pub/firefox/releases/$FIREFOX_VERSION.0esr/linux-x86_64/en-US/firefox-$FIREFOX_VERSION.0esr.tar.bz2 \
+ADD https://download-installer.cdn.mozilla.net/pub/firefox/releases/$FIREFOX_VERSION/linux-x86_64/en-US/firefox-$FIREFOX_VERSION.tar.bz2 \
```
Comment (mshriver) [inline Dockerfile.stable:32]: :+1:

File: Dockerfile.stable [hunk 3]
```diff
@@ -20,15 +24,20 @@ RUN mkdir -p /root/chrome-driver &&\
...
     /root/firefox.tar.bz2
+ADD https://github.com/mozilla/geckodriver/releases/download/$GECKODRIVER_VERSION/geckodriver-$GECKODRIVER_VERSION-linux64.tar.gz \
+    /root/gecko.tar.gz
+
 RUN tar -C /root/ -xjvf /root/firefox.tar.bz2 && rm -f /root/firefox.tar.bz2
+RUN tar -C /root/firefox/ -xvf /root/gecko.tar.gz && rm -f /root/gecko.tar.gz
 
 # runtime
-EXPOSE 22
```
Comment (mshriver) [inline Dockerfile.stable:31]: No more SSH?

File: Dockerfile.stable (hunk 1 above)
Comment (RonnyPfannschmidt) [reply Dockerfile.stable:28]: the cutoff is incorrect, so this is a partial solution eventually there will be a major/minor + a patchlevel var

File: Dockerfile.stable (hunk 3 above)
Comment (RonnyPfannschmidt) [reply Dockerfile.stable:31]: wasn't started to begin with, so this lowers port cost

**Pull request merged**